administrator/language/en-GB/com_templates.ini

```
@@ -37,6 +37,8 @@ COM_TEMPLATES_CONFIG_SOURCE_LABEL="Valid Source Formats"
 COM_TEMPLATES_CONFIG_UPLOAD_LABEL="Max. Upload Size (MB)"
 COM_TEMPLATES_CONFIGURATION="Template: Options"
 COM_TEMPLATES_COPY_SUCCESS="New template called %s was installed."
+JTOOLBAR_CREATE_CHILD="Duplicate as child"
+JTOOLBAR_OVERRIDE_CHILD="Manage overrides"
```

As per the style guide https://developer.joomla.org/en-gb-user-interface-text-guidelines/capitalisation.html

```suggestion
JTOOLBAR_CREATE_CHILD="Duplicate as Child"
JTOOLBAR_OVERRIDE_CHILD="Manage Overrides"
```